lib/iris/fileformats/__init__.py

```diff
@@ -28,12 +28,9 @@
 from . import abf
 from . import um
 try:
-    import iris_grib as igrib
+    from . import grib as igrib
```

Comment:
I don't think this try/except is necessary. Presumably the `grib` sub-module will always be available now?